.travis.yml

```diff
@@ -11,6 +11,13 @@ before_install:
   # Needed to deploy pull request and releases
   - sudo apt-get -y install python-pip python-dev
   - pip install awscli --upgrade --user
+  # use production url for price estimator on master or tagged
+  - export PRICE_ESTIMATOR_URL=$([[
+    $TRAVIS_BRANCH = master || \
+    $TRAVIS_BRANCH = release/* || \
+    $TRAVIS_BRANCH = hotfix/* || \
```

Comment:
Great! I guess to make sure it works, we need to merge the thing :) 

We now have an open hotfix, it's a great moment to get this thing merged 👍